broker-core/src/test/java/org/apache/qpid/server/security/group/FileGroupDatabaseTest.java

```diff
@@ -215,21 +217,14 @@ public void testGetGroupPrincipalsForUserWhenUserRemovedFromGroup() throws Excep
         assertTrue(_fileGroupDatabase.getGroupsForUser(USER1.toUpperCase()).isEmpty());
     }
 
-    @Test
+    @Test(expected = IllegalConfigurationException.class)
     public void testGetGroupPrincipalsForUserWhenUserAddedToGroupTheyAreAlreadyIn() throws Exception
     {
         _util.writeAndSetGroupFile("myGroup.users", USER1);
         _fileGroupDatabase.addUserToGroup(USER1, MY_GROUP);
```

Comment:
the test throws exception at this line
the rest of the test is a dead code... please remove code after this line

Reply:
Done!